src/App/App.test.js

```diff
@@ -0,0 +1,66 @@
+import { screen, render, waitFor } from '@testing-library/react'
+import App from './App.js'
+import '@testing-library/jest-dom'
+import { getAllMovies, getCurrentMovie } from '../apiCalls.js'
+import userEvent from '@testing-library/user-event'
+jest.mock('../apiCalls.js')
+
+describe('App', () => {
+
+  beforeEach(() => {
+
+    getAllMovies.mockResolvedValueOnce({movies: [{
```

Comment:
Remember for future reference that what data type is being fetched so that it can be mocked during tests